cssom-view-1/Overview.bs

```diff
@@ -1908,15 +1907,23 @@ Each {{Document}} has an associated list of <dfn for=Document>pending scroll eve
 
 Each {{Document}} has an associated list of <dfn for=Document>pending scrollend event targets</dfn>, initially empty.
 
+Each {{Document}} has an associated list of <dfn export for=Document>pending snapchanging event targets</dfn>, initially empty.
+
+Each {{Document}} has an associated list of <dfn export for=Document>pending snapchanged event targets</dfn>, initially empty.
```

Comment:
I think it might be good to move everything over to scroll-snap-2 and only have cssom-view invoke methods over in scroll-snap-2 at the appropriate times rather than having some of the algorithm in each place.